deployments/pulse3d/services/pulse3d/src/main.py

```diff
@@ -35,8 +35,9 @@ class UploadResponse(BaseModel):
 
 class JobRequest(BaseModel):
     upload_id: uuid.UUID
-    prominence_factors: Optional[Union[Tuple[Union[int, float]], List[int]]]
-    width_factors: Optional[Union[Tuple[Union[int, float]], List[int]]]
+    prominence_factors:Optional[Union [Tuple [Union [int,float]], List[Union[int,float]],int,float]]
```

Comment:
Since we don't have pre-commit in this repo it might be good to install black to format python files and prettier to format js files. If you're using VSCode you can install both of these as VSCode extensions and configure them to format on save